config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-periodics.yaml

```diff
@@ -0,0 +1,31 @@
+periodics:
+- name: periodic-cluster-api-provider-digitalocean-conformance-v1alpha3
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  interval: 12h
```

Comment:
Since we don't have a lot of traffic in our repo, should we reduce this a bit? Maybe 24 hours instead?

Reply:
sound good

Reply:
done